internal/sys/fd.go

```diff
@@ -81,10 +81,13 @@ func (fd *FD) Close() error {
 		return nil
 	}
 
-	return unix.Close(fd.disown())
+	return unix.Close(fd.Disown())
 }
 
-func (fd *FD) disown() int {
+// Disown destroys the FD and returns its raw file descriptor without closing
```

Comment:
Nittiest of nits: This comment gets added with the last commit instead of the first.

Reply:
MVP comment, looks like I messed up the fixup, thanks!